meson.build

```diff
@@ -1,8 +1,8 @@
-project('PackageKit', 'c',
+project('PackageKit', 'c', 'cpp',
```

Comment:
This needs to be in the backend's `meson.build`, not here.

Reply:
Thanks, done.